tests/integration/targets/win_acl_inheritance/tasks/main.yml

```diff
@@ -2,14 +2,13 @@
 # Test setup
 # Use single task to save in CI runtime
 - name: create test folders
-  win_shell: |
+  ansible.windows.win_shell: |
```

Comment:
Just an FYI you don't need to use the fully qualified name for things in this collection. You don't need to change it back but I'm unsure if you did this manually for some reason or it was changed due to another linting tool.

Reply:
Just a habit :) After the linter recommendations, it became a habit for me, I use them in all my roles and playbooks.

Reply:
Fair enough, I assumed it was something like that.